### DISCUSSION TITLE
Confused about variants.ini options/versions

### DISCUSSION THREAD

**Discussion by rapiermother** [fairy-stockfish/Fairy-Stockfish#666] [Q&A] [answered]
I'm having trouble implementing some of the variants.ini options, as Fairy-Stockfish doesn't recognize them.

I've followed the directions on the [site](https://github.com/fairy-stockfish/Fairy-Stockfish), using both the "[Windows Executable](https://github.com/fairy-stockfish/Fairy-Stockfish/releases/latest/download/fairy-stockfish-largeboard_x86-64.exe)" link and the "[latest release](https://github.com/fairy-stockfish/Fairy-Stockfish/releases/latest)" link, and neither executable (when running check variants.ini) recognizes a handful of the options.

I've noticed that the [variants.ini](https://github.com/fairy-stockfish/Fairy-Stockfish/releases/download/fairy_sf_14/variants.ini) linked to on the latest release page is significantly different than the [variants.ini](https://github.com/ianfab/Fairy-Stockfish/blob/master/src/variants.ini) linked to directly on the site. It seems that the more prominent (read: site) link to variants.ini simply has more options, but this implies that there is a newer release of Fairy-Stockfish capable of working with these options.

I'm wondering if I'm just missing another link somewhere on the site that will point me to this newer version, or if there is another explanation and/or method of implementing such options as castlingRookKingsideFile, pawnTypes, and others in my variants.

**Comment by ianfab** [comment, marked as answer]
Hi. The [latest variants.ini](https://github.com/ianfab/Fairy-Stockfish/blob/master/src/variants.ini) is the one compatible to the current development version. Therefore in order to use that you need to download a recent development version from the [github actions](https://github.com/fairy-stockfish/Fairy-Stockfish/actions/workflows/release.yml?query=branch%3Amaster+event%3Apush). If you want to use official releases you need to use the corresponding variants.ini of that release.

By using a development version you can use new features earlier, but until there is an official release with the new features they might still be subject to frequent change.

**Reply by rapiermother** [reply]
I can confirm empirically that this is the correct solution. I really appreciate your guidance.
